### ISSUE TITLE
How to allow tags

### ISSUE THREAD

**Issue by neomusic** [SamuelBagattin/terraform-aws-github-oidc-provider#5]
I would like to be able to set, in addition to the `allowed_branches`, also the tags. 

How can I do that?

for example, i want this similar output:
```
"Condition": {
                "StringLike": {
                    "token.actions.githubusercontent.com:sub": "repo:foo/bar:ref:*"
                }
  }
```

**Comment by SamuelBagattin** [comment]
Thanks for opening this issue.
The module does not currently support this feature.
The subject condition must be `refs/tags/xxxx` instead of `refs/heads/xxx` according to [the official documentation](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#filtering-for-a-specific-tag).
An edit can be made in the [locals.tf file](https://github.com/SamuelBagattin/terraform-aws-github-oidc-provider/blob/main/locals.tf#L40) and by adding a new variable for the tags list.

**Comment by SamuelBagattin** [comment]
@neomusic the tag feature has just beef released in the 0.4.0 version
